platforms/linux-s390x/Dockerfile

```
@@ -35,6 +35,7 @@ RUN \
     --no-modify-path \
     --profile minimal \
     && \
+  rustup default nightly && \
   rustup target add s390x-unknown-linux-gnu && \
```

```suggestion
    --profile minimal \
    --target s390x-unknown-linux-gnu \
    --default-toolchain nightly \
    && \
```
(I also changed it to use `--target ...`  instead of `rustup target add ...`)